docs/astro/sql/operators/drop_table.rst

```diff
@@ -1,9 +1,9 @@
+.. _drop_table_operator:
+
 ======================================
-drop operator
+drop table operator
 ======================================
 
-.. _drop_operator:
-
 When to Use drop operator
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 The drop operator is used to delete tables from the database if they exist. It can be used on both Temporary as well as Persistent Tables.
```

Comment:
```suggestion
The drop table operator is used to delete tables from the database if they exist. It can be used on both Temporary as well as Persistent Tables.
```